### PR TITLE
Fix readme

### LINKED ISSUE #1075: Update core readme
**Describe the bug**

- Should update java version required to 17.
- Should add `geojson-pydantic` requirement

### PR DESCRIPTION
**Pull request by flomonster**
close #1075

### CODE REVIEW

File: core/README.md [hunk 1]
```diff
@@ -39,8 +40,9 @@ You'll need:
...
 
-java -jar build/libs/osrd-all.jar \
-    simulate \
-    --config build/resources/test/tiny_infra/config_railjson.json \
-    -o sim_changelog_output.json
+# Run as service
+java -jar build/libs/osrd-all.jar api -p 8080
+
+# Check that an infra can be loaded
+java -jar build/libs/osrd-all.jar load-infra --path RAILJSON_INFRA
```
Comment (elzinko) [inline core/README.md:47]: What about specifying RAILJSON_INFRA before ?

**Comment by elzinko** [comment]
@flomonster tests failed

**Review by elzinko — APPROVED** [review]

File: core/README.md (hunk 1 above)
Comment (flomonster) [reply core/README.md:47]: `RAILJSON_INFRA` is the value of the option `--path`

File: api/README.md [hunk 2]
```diff
@@ -1,91 +1,31 @@
-# osrd
+# OSRD's API
 
-## Using the API
+This service handle OSRD models and schema using [Django](https://www.djangoproject.com/).
+Multiple extension are used such as:
+  - [Django Rest Framework](https://www.django-rest-framework.org/)
+  - [Django Rest Framework Gis](https://github.com/openwisp/django-rest-framework-gis)
+  - [Pydantic](https://pydantic-docs.helpmanual.io/)
```
Comment (elzinko) [inline api/README.md:7]: is gdal mandatory ?

**Pull request merged**